build-tools/packages/build-tools/src/fluidBuild/fluidRepoBuild.ts

```diff
@@ -32,8 +32,17 @@ export interface IPackageMatchedOptions {
 }
 
 export class FluidRepoBuild extends FluidRepo {
-	constructor(resolvedRoot: string) {
-		super(resolvedRoot);
+	public static create(resolvedRoot: string) {
```

Comment:
What's the advantage of static create vs. constructor?

Reply:
Just stylistic.  I am used to have "super" call be always the first thing in a constructor.

Also, it will mirror FluidRepo creation pattern.